src/engine/source/builder/builders/opBuilderCondition.cpp

```diff
@@ -19,53 +20,145 @@
 
 namespace builder::internals::builders
 {
+std::function<bool(types::Event)>
+middleBuilderConditionReference(const types::DocumentValue& def,
+                                types::TracerFn tr)
+{
+    if (!def.MemberBegin()->name.IsString())
+    {
+        throw std::runtime_error("Error building condition reference, key of "
+                                 "definition must be a string.");
+    }
+    if (!def.MemberBegin()->value.IsString())
+    {
+        throw std::runtime_error("Error building condition reference, value of "
+                                 "definition must be a string.");
+    }
+
+    // Estract and prepare field and reference
+    std::string field {
+        json::formatJsonPath(def.MemberBegin()->name.GetString())};
+    std::string reference {def.MemberBegin()->value.GetString()};
```

Comment:
Please, make them "const"

Reply:
only made const field, reference is edited just in the following line